src/utils.ts

```diff
@@ -72,19 +74,27 @@ export const extractParamsFromBody = (body: string) => {
  * @param project The project namespace, i.e.: retail-pos-web
  * @param resource The resource name, i.e.: retail-reports
  * @param languages The languages to pull, i.e.: ['fr', 'de']
+ * @param mode "translated" | "reviewed" | "proofread"
+ * @param branch Base branch to pull changes from reading the correct .tx/config
  */
 export const pullTranslations = async (
   project: string,
   resource: string,
   languages: string[],
-  mode: InputMode
+  mode: InputMode,
+  branch: string
```

Comment:
You can default this to `master` so that this is a non-breaking change for web-platform.

Reply:
this `pullTranslations` didn't need master because that's the branch already checked out by the `uses: actions/checkout@v2`, but it needed the default `master` in the `createPullRequest` function